release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1810,6 +1810,139 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Previously, when a compute node was trying to join a cluster the rendezvous node rebooted before the process completed. As the compute node could not communicate as expected with the rendezvous node, the installation was not successful. With this release, a patch is applied that fixes the racing condition that caused the rendezvous node to reboot prematurely and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41811[*OCPBUGS-41811*])
+
+* Previously, when using the {ai-full}, selecting a multi-architecture image for `s390x` CPU architecture on {hybrid-console} could cause the installation to fail. The installation program reported an error that the new cluster was not created because the skip MCO reboot was not compatible with `s390x` CPU architecture. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41716[*OCPBUGS-41716*])
+
+* Previously, a coding issue caused the Ansible script on {rh-openstack} user-provisioned infrastructure installation to fail during the provisioning of compact clusters. This occurred when IPv6 was enabled for a three-node cluster. With this release, the issue is resolved and you can provision compact three-node clusters. (link:https://issues.redhat.com/browse/OCPBUGS-41538[*OCPBUGS-41538*])
+
+* Previously, a coding issue caused the Ansible script on {rh-openstack} user-provisioned installation infrastructure to fail during the provisioning of compact clusters. This occurred when IPv6 was enabled for a three-node cluster. With this release, the issue is resolved and you can provision compact three-node clusters
+on {rh-openstack} for user-provisioned installation infrastructure. (link:https://issues.redhat.com/browse/OCPBUGS-39402[*OCPBUGS-39402*])
+
+* Previously, the order of an Ansible Playbook was modified to run before the `metadata.json` file was created, which caused issues with older versions of Ansible. With this release, the playbook is more tolerant of missing files to accommodate older versions of Ansible and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-39285[*OCPBUGS-39285*])
+
+* Previously, when you installed a cluster there were issues using a compute node because the date, time, or both might have been inaccurate. With this release, a patch is applied to the live ISO time synchronization. The patch configures the `/etc/chrony.conf` file with the list of the additional Network Time Protocol (NTP) servers that the user provides in the `agent-config.yaml` file, so that you can use a worker node without experiencing a cluster installation issue. (link:https://issues.redhat.com/browse/OCPBUGS-39231[*OCPBUGS-39231*])
```

Comment:
I missed the second instance of "worker" in this one when i reviewed - sorry!
```suggestion
* Previously, when you installed a cluster there were issues using a compute node because the date, time, or both might have been inaccurate. With this release, a patch is applied to the live ISO time synchronization. The patch configures the `/etc/chrony.conf` file with the list of the additional Network Time Protocol (NTP) servers that the user provides in the `agent-config.yaml` file, so that you can use a compute node without experiencing a cluster installation issue. (link:https://issues.redhat.com/browse/OCPBUGS-39231[*OCPBUGS-39231*])
```